apps/sample-workers-vite-react/README.md

```diff
@@ -0,0 +1,48 @@
+# Xata Workers Vite React Sample App
+
+This is a sample app that uses [Xata Workers](https://xata.io/beta/workers) and [Vite](https://vitejs.dev) to build a React app.
+
+## Getting Started
+
+1. Install dependencies
+
+```bash
+pnpm install
+```
+
+2. Connect to your Xata database (demo db)
+
+```bash
+xata init
+```
+
+3. Configure Xata Workers
+
+```bash
+xata workers init
+```
```

Comment:
<img width="414" alt="image" src="https://user-images.githubusercontent.com/1761469/204254525-d479806e-2b9a-4b69-961e-c38ee2f6e955.png">

With `@xata.io/cli/0.11.1`

Reply:
Going out in a few minutes in `0.11.2`: https://github.com/xataio/client-ts/pull/790

Reply:
Shouldn't we bump a minor because it's a new feature?